File: ios/Classes/FaceLivenessDetectionCallbacks.swift
typealias FaceLivenessDetectionCallback = (UIImage, String) -> Void

typealias FaceLivenessDetectionStatusUpdateCallback = (Int) -> Void

typealias FaceLivenessDetectionErrorCallback = (String) -> Void

typealias FaceLivenessDetectionStartedCallback = (FaceLivenessDetectionStartParameters) -> Void


struct FaceLivenessDetectionStartParameters {
    var width: Double = 0.0
    var height: Double = 0.0
    var textureId: Int64 = 0
}

File: android/src/main/kotlin/com/kbtg/face_liveness_detection/FaceLivenessDetectionCallbacks.kt
package com.kbtg.face_liveness_detection

import android.graphics.Bitmap
import com.kbtg.face_liveness_detection.objects.FaceLivenessDetectionStartParameters

typealias FaceLivenessDetectionCallback = (image: Bitmap, scores: String) -> Unit
typealias FaceLivenessDetectionStatusUpdateCallback = (state: Int) -> Unit
typealias FaceLivenessDetectionErrorCallback = (error: String) -> Unit
typealias FaceLivenessDetectionStartedCallback = (parameters: FaceLivenessDetectionStartParameters) -> Unit
